CONTRIBUTING.md

```diff
@@ -6,18 +6,33 @@ Learn the rules to follow when you contribute code or content to this project.
 
 ## Contributing process
 
-The contributing process in this project relies on the [GitHub flow](https://guides.github.com/introduction/flow/index.html). This means that you contribute through pull requests. When adding new code or content to this project, follow these rules:
+The contributing process in this project relies on the [GitHub flow](https://guides.github.com/introduction/flow/index.html). This means that you contribute through pull requests (PRs). When adding new code or content to this project, follow these rules:
 
 1. Fork this repository.
-2. Make your changes. Do not forgot about:
+1. Make your changes. Do not forgot about:
    - [Naming and architecture convention](./DEVELOPMENT.md#naming--architecture-convention)
    - [Project structure](./development-guide.md/#project-structure)
    - Tests
    - Updating relevant documents if you add, remove, update props, configuration, or themes
-3. Squash your changes to a single commit. Write a concise commit message in the imperative mood, as described [here](https://chris.beams.io/posts/git-commit/).
-4. Rebase your changes to the latest `master` branch.
-5. Create a pull request.
+1. Create a PR.
 
 > **NOTE:** When you add content to the `converter-go` repository, follow the [guidelines](https://github.com/kyma-project/community/tree/master/guidelines/content-guidelines) from the `kyma-project` organization.
 
-Pull requests are very welcome. However, if you want to add a new feature, use GitHub issues to discuss your ideas first.
\ No newline at end of file
+PRs are very welcome. However, if you want to add a new feature, use GitHub issues to discuss your ideas first.
+
+## Conventional commits
+
+This project follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification. Releasing to GitHub and NPM is done with the support of [semantic-release](https://semantic-release.gitbook.io/semantic-release/).
+
+A PR should have a title that follows the specification, otherwise, merging is blocked. If you are not familiar with the specification simply ask maintainers to modify the PR. You can also use this cheatsheet:
```

Comment:
```suggestion
A PR should have a title that follows the specification, otherwise, merging is blocked. If you are not familiar with the specification, simply ask maintainers to modify the PR. You can also use this cheatsheet:
```